src/shared/js/components/Popup/Popup.js

```diff
@@ -40,7 +39,7 @@ function Popup() {
         <div>Is Enabled: {sideKickEnabled ? 'Yes' : 'No'}</div>
       </main>
       <footer>
-        <span className="version">v0.0.9</span>
```

Comment:
How are we deciding what the version number is? May be usefull to keep it as 0.0.x for now, since this is still in development and a 0.0.x version allows us to make breaking changes. @bradtaylorsf should probably weigh in on this.

Reply:
True, and I wonder if that can be grabbed automatically so we don't have to manually update it

Reply:
try this: const { version } = require('../path/to/package.json');

Reply:
Thanks!! Works like a charm.